ert/serialization/_serializer.py

```diff
@@ -31,6 +31,8 @@ async def decode_from_path(
 
 class _json_serializer(Serializer):
     def encode(self, obj: Any, *args: Any, **kwargs: Any) -> str:
+        if "indent" not in kwargs:
```

Comment:
Is it possible to move this into the function declaration as an optional argument and then afterwards passed explicitly to `dumps`. It should yield the same behaviour and maybe look a bit less wacky :)

Reply:
Yes, that worked! Looks better now.

Reply:
Nope, `pylint` complained:
```
************* Module ert.serialization._serializer
ert/serialization/_serializer.py:33:4: W0221: Number of parameters was 4 in 'Serializer.encode' and is now 5 in overridden '_json_serializer.encode' method (arguments-differ)

```

So either go wacky, add pyllint exception, or add `indent` as a argument to all `encode()` implementations.